src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -175,6 +175,12 @@ public IDisposable On(string methodName, Type[] parameterTypes, Func<object[], o
 
             return new Subscription(invocationHandler, invocationList);
         }
+        
+        public void Off(string methodName)
```

Comment:
I'm ok with `Off`.

Should there be an overload that takes a `handler` to remove a specific instance?

Reply:
Also, need XML docs.

Reply:
>Should there be an overload that takes a handler to remove a specific instance?

I didn't add it on this pass because we already have  a way to remove a specific handlers. Disposing the associated subscription.